lib/handlers/QuizAnswersHandler.js

```diff
@@ -0,0 +1,30 @@
+const util = require('../util');
+
+/**
+ *
+ * @param connection
+ * @param data
+ * @param storage
+ * @returns {Promise<any>}
+ */
+async function quizAnswersHandler (connection, data, storage) {
+    console.log("Recieved answers");
+    let successful = false;
+    let dataView = new Uint8Array(data);
+    let answers = {
+        index: dataView[1],
+        answers: Array.from(dataView.slice(1))
+    };
+
+    //todo validate answers (dependent on quiz object being implemented)
```

Comment:
Can't implement some of the validation because I don't know what a quiz object is going to look like

Reply:
Sounds good, we can do the actual implementation later.